pkg/util/test/util.go

```diff
@@ -3015,8 +2994,8 @@ func ValidateTelemetryV2Enabled(pxImageList map[string]string, cluster *corev1.S
 			return nil, true, err
 		}
 
-		if !validateTelemetryHealthyInPxctl(cluster.Namespace) {
-			return nil, true, fmt.Errorf("failed to validate that Telemetry is Healthy in pxctl status")
+		if err := validateTelemetryStatusInPxctl(true, cluster.Namespace); err != nil {
```

Comment:
because this function has builtin retry loop, we can move it out of this wrap and put it after those component validation. it also makes sense to check pxctl status after all components got validated.

Reply:
Done